pkg/provisioner/block/block.go

```diff
@@ -70,22 +71,33 @@ func roundUpSize(volumeSizeBytes int64, allocationUnitBytes int64) int64 {
 	return (volumeSizeBytes + allocationUnitBytes - 1) / allocationUnitBytes
 }
 
-// Provision creates an OCI block volume acording to the spec
-func (block *blockProvisioner) Provision(options controller.VolumeOptions,
-	availabilityDomain *identity.AvailabilityDomain) (*v1.PersistentVolume, error) {
+// Provision creates an OCI block volume
+func (block *blockProvisioner) Provision(options controller.VolumeOptions, availabilityDomain *identity.AvailabilityDomain) (*v1.PersistentVolume, error) {
 	for _, accessMode := range options.PVC.Spec.AccessModes {
 		if accessMode != v1.ReadWriteOnce {
-			return nil, fmt.Errorf("invalid access mode %v specified. Only %v is supported",
-				accessMode,
-				v1.ReadWriteOnce)
+			return nil, fmt.Errorf("invalid access mode %v specified. Only %v is supported", accessMode, v1.ReadWriteOnce)
 		}
 	}
 
-	// Calculate the size
-	volSize := options.PVC.Spec.Resources.Requests[v1.ResourceName(v1.ResourceStorage)]
-	volSizeBytes := volSize.Value()
-	glog.Infof("Volume size (bytes): %v", volSizeBytes)
-	volSizeMB := int(roundUpSize(volSizeBytes, 1024*1024))
+	// Calculate the volume size
```

Comment:
Would probably split this out into a method/fn.

Reply:
Agree but will do in a subsequent PR to tidy up this section more generally.